### PR TITLE
Basic UI fonts

### LINKED ISSUE #3523: [BasicUI] Icons & font MaterialIcons-Regular are missing
In BasicUI some of the icons do not show up, i.e. the group navigation (arrow left/right) or the color picker items. In addition the requested font MaterialIcons-Regular is not available from the web/fonts folder. I wonder why this ever worked because the files for this font never were part of the repository according to git history.

image

I can see that @resetnow did changes to material-icons.css which indicate that this should have been working some day with 072e76c7be6efadd93821fba17b82516b3265c83 what am I missing here?

When I revert 072e76c7be6efadd93821fba17b82516b3265c83 and put current version of MaterialIcons-Regular.woff2 in the web/fonts folder everything works well.

Tested in Chrome/Safari/Firefox on macOS.

### PR DESCRIPTION
**Pull request by sjka**
This PR includes the material iconset in Basic UI in the same way as in the Paper UI. 

It uses NPM to download the fonts and copies them to the corresponding folder. As before, I also checked them in so that the Basic UI works out-of-the-box and developers aren't required to run a maven build first. That's how it is today and a didn't want to change that paradigm in this PR.

As discussed in #3523, this PR also includes a revert-commit of 072e76c in order to adapt the icons to our approved version of the font. 

Note to openHAB: The fragment needs to be adapted to not include the material icon font anymore, as the versions are not compatible.

fixes #3523
Signed-off-by: Simon Kaufmann <simon.kfm@googlemail.com>

### CODE REVIEW

**Comment by sjka** [comment]
@resetnow would you mind having a look if what I did makes somewhat sense?

**Comment by maggu2810** [comment]
> Note to openHAB: The fragment needs to be adapted to not include the material icon font anymore, as the versions are not compatible.

@kaikreuzer I leave the merge up to you, so you can do a proper synchronization of that tasks.

**Comment by kaikreuzer** [comment]
Looks better now.

File: extensions/ui/org.eclipse.smarthome.ui.basic/gulpfile.js [hunk 1]
```diff
@@ -13,6 +13,20 @@
 			sass: "web-src/smarthome.scss"
 		};
 
+	
+	var paths = {
+	        FontLibs: [
```
Comment (resetnow) [inline extensions/ui/org.eclipse.smarthome.ui.basic/gulpfile.js]: `fontLibs`

**Comment by sjka** [comment]
Argh! Thanks, I will fix this.

**Pull request merged**